src/packages/hex.gleam

```diff
@@ -1,55 +1,180 @@
 import gleam/io
 import gleam/json
-import gleam/http
+import gleam/http/request
+import gleam/http.{Get}
 import gleam/result
 import gleam/hackney
-import gleam/dynamic.{Decoder} as d
+import gleam/list
+import gleam/dynamic as d
+import time/parse.{parse_iso8601_to_gregorian_seconds}
+import gleam/pgo
+import gleam/string
+import gleam/int
+import packages/models/hex/package.{HexPackage, hex_package_decoder}
+import packages/models/hex/release.{hex_release_decoder}
 
 pub type Error {
   HackneyError(hackney.Error)
   JsonError(json.DecodeError)
+  DatabaseError(pgo.QueryError)
+  ItemNotInListError(Nil)
```

Comment:
This error doesn't explain what has happened, let's remove it or replace it with a higher level error

Reply:
Changed to a more descriptive error